src/transport/p2p.cc

```diff
@@ -115,7 +115,7 @@ ncclResult_t p2pCanConnect(int* ret, struct ncclTopoSystem* topo, struct ncclTop
     // Check that legacy IPC support is available (WSL WAR)
     char *dummy;
     hipIpcMemHandle_t ipc;
-    NCCLCHECK(ncclCudaCalloc(&dummy, CUDA_IPC_MIN));
+    CUDACHECK(hipMalloc(&dummy, CUDA_IPC_MIN));
```

Comment:
why this is changed?

Reply:
This dummy memory is immediately freed after this.  It doesn't need to be calloc-ed.